### PR TITLE
Adding support for politeFocus

### PR DESCRIPTION
**Pull request by samuelfreiberg**
Currently, there is no exposed mechanism for "Polite Focus" exposed from ReactComponent NetUI elements. This PR will add support from the JS side for Polite Focus. The native changes for this are almost complete as well. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6583)

### CODE REVIEW

**Comment by NickGerleman** [comment]
ViewWin32 doing custom stuff doesn't have some of the consequences of forking, but it would be great to converge NetUI needs with stuff other platforms are doing. There's a lot of discussion happening right now about what desktop focus should look like across the platform and it would be great to contribute this go that conversation. 

@rectified95 might have some interest in this.

File: packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx [hunk 1]
```diff
@@ -81,12 +81,23 @@ export const ViewWin32 = React.forwardRef(
...
-              findNodeHandle(localRef),
-              NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.focus,
-              null
-              );
+          localRef.focus = (usePoliteFocus?: boolean) => {
+            if(usePoliteFocus)
+            {
+              NativeModules.UIManager.dispatchViewManagerCommand(
+                findNodeHandle(localRef),
+                NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.politefocus,
```
Comment (jonthysell) [inline packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx:89]: Are you missing files? Where is 'politeFocus' defined?

**Comment by asklar** [comment]
@samuelfreiberg looks like a bad merge (1077 changed files / 284 commits) - can you please rebase/resolve?

**Comment by NickGerleman** [comment]
There was a whole thing that happened here. Will send and email on Monday.

I first ran into the issue of repo admin privileges meaning commits can go directly to master if you hit "push" instead of "push to" in VSCode. Since I caught it quickly, I went to force push the previous master branch without the accidental commits.

I initially force pushed the wrong branch... We're back to master being in the right state now (same commit hashes and all), pre accidental commits, but the initial force push to an older branch seems to have changed PR state, where the second didn't catch things up.

Moral of the story: Don't force push to master, even if you think you know what you're doing.

**Comment by NickGerleman** [comment]
Merged all open PRs to the latest commit in the master branch where they don't have conflicts, so everything should show up correctly again.

File: packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx (hunk 1 above)
Comment (NickGerleman) [reply packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx]: Adding some context, react-native-win32 is the package the JS portion of the react-native platform for NetUI. Its view managers and most of its native modules live in the Office devmain repo (though some are shared and live in this repo). 

This is organizationally pretty funky, and not the long term repo we want this to be in. Having it live here ends up being dramatically better than alternatives ease of maintenance wise, and has helped us spend more time on impactful problems for RNW.

File: packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx [hunk 2]
```diff
@@ -86,8 +86,24 @@ export const ViewWin32 = React.forwardRef(
               findNodeHandle(localRef),
               NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.focus,
               null
-              );
+              );
```
Comment (asklar) [inline packages/react-native-win32/src/Libraries/Components/View/ViewWin32.tsx:89]: nit looks like extra space got added, are these not being linted?

**Review by asklar — APPROVED** [review]

**Pull request closed without merging**